pkg/agent/openflow/network_policy.go

```diff
@@ -944,6 +953,21 @@ func (c *client) UninstallPolicyRuleFlows(ruleID uint32) error {
 		return err
 	}
 
+	isFound := false
+	for _, ctxChange := range ctxChanges {
```

Comment:
ctxChanges seem to be some what complicated.

Is it possible to change following function in reconciler.go to accept Direction or pointer to rule as installOFRule? 
func (r *reconciler) uninstallOFRule(ofID uint32)

Reply:
Thank you for your suggestion, let me take a look into it.